app/pages/admin/user-settings.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+
+import AutoSave from '../../components/auto-save';
+import handleInputChange from '../../lib/handle-input-change';
+import UserLimitToggle from './user-settings/limit-toggle';
+
+class UserSettings extends Component {
+  constructor(props) {
+    super(props);
+    this.boundForceUpdate = this.forceUpdate.bind(this);
+  }
+
+  componentDidMount() {
+    if (this.props.editUser != null) {
+      this.props.editUser.listen('change', this.boundForceUpdate);
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.editUser != null) {
+      this.props.editUser.stopListening('change', this.boundForceUpdate);
+    }
+    nextProps.editUser.listen('change', this.boundForceUpdate);
+  }
+
+  render() {
+    if (!this.props.editUser) {
+      return (
+        <div>No user found</div>
+      );
+    }
+
+    const handleChange = handleInputChange.bind(this.props.editUser);
+
+    return (
+      <div className="project-status">
+        <h4>Settings for {this.props.editUser.login}</h4>
+        <ul>
+          <li>
+            <AutoSave resource={this.props.editUser}>
+              <input type="checkbox" name="admin" checked={this.props.editUser.admin} disabled onChange={handleChange} />{' '}
+              Admin
```

Comment:
`admin`, `login_prompt`, and `private_profile` were all disabled before this PR -- I just re-implemented what was there.  Happy to remove the `AutoSave` and `onChange`.  You're call